rest/access-tokens/ip-messaging-example/ip-messaging-example.5.x.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Twilio.JWT;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // These values are necessary for any access token
+        const string twilioAccountSid = "ACxxxxxxxxxxxx";
```

Comment:
What about keep consistency in the usage of `accountSid`. In most of the examples available we have this format:

```csharp
const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
```

Having a consistent format will make easier for the reader to understand the documentation. Any inconsistent formatting is usually unfriendly. What do you think?